actions/actions.py

```diff
@@ -167,9 +205,10 @@ async def explain_credit_card(
     ) -> Dict[Text, Any]:
         """Explains 'credit_card' slot"""
```

Comment:
I am getting an error for this conversation:

```bash
(financial-demo-db) KUDU arjaan (db) financial-demo-2 $ rasa shell
2021-01-19 08:39:49 INFO     rasa.model  - Loading model models/20210119-080710.tar.gz...
2021-01-19 08:39:50 INFO     root  - Connecting to channel 'cmdline' which was specified by the '--connector' argument. Any other channels will be ignored. To connect to all given channels, omit the '--connector' argument.
2021-01-19 08:39:50 INFO     root  - Starting Rasa server on http://localhost:5005
2021-01-19 08:39:50 INFO     rasa.model  - Loading model models/20210119-080710.tar.gz...
2021-01-19 08:40:10 INFO     rasa.nlu.components  - Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en_core_web_md'.
2021-01-19 08:40:12 INFO     root  - Rasa server is up and running.
Bot loaded. Type a message and press enter (use '/stop' to exit): 
Your input ->  pay credit card                                                                                                 
Towards which credit card account do you want to make a payment?
Your input ->  visa                                                                                                            
Sorry, that is not a valid credit card account to make payments towards.
Towards which credit card account do you want to make a payment?
Your input ->  emblem                                                                                                          
2021-01-19 08:40:56 ERROR    rasa.core.nlg.interpolator  - Failed to fill utterance template 'The current balance for your {credit_card} account is {currency}{credit_card_balance}.'. Tried to replace 'credit_card_balance' but could not find a value for it. There is no slot with this name nor did you pass the value explicitly when calling the template. Return template without filling the template. 
Traceback (most recent call last):
  File "/home/arjaan/miniconda3/envs/financial-demo-db/lib/python3.7/site-packages/rasa/core/nlg/interpolator.py", line 28, in interpolate_text
    text = text.format(values)
KeyError: 'credit_card_balance'
```

Reply:
Fixed the utterance by updating utterance in domain.yml, using `amount-of-money` instead of `credit_card_balance`.

Looking into reason why it is not accepting emblem as a valid credit card in this conversation.

Reply:
@melindaloubser1 ,
found it!
The new logic makes only a subset of cards available for each session:

```bash
session_credit_cards = sample(
            credit_card_names, choice(list(range(2, len(credit_card_names))))
        )
```

Reply:
Ah yes, I wondered about that. We can change it to be all cards always.

Reply:
Yeah, I think it is better to use all cards always, to make the behavior predictable.
Also important for demo purposes to have a consistent behavior.

Reply:
I pushed that update.